hack/check-pr.sh

```diff
@@ -21,24 +21,8 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-dirs=()
-tests=()
-ref="${1:-HEAD}"
-echo -n "Packages changed since $ref: "
-for d in $(git diff --name-only "$ref" | xargs -n 1 dirname | sort -u); do
-    if ! ls "./$d/"*.go &> /dev/null; then
-        continue
-    fi
-    echo -n "$d "
-    dirs+=("./$d")
-    tests+=("//$d:all")
-done
-
-if [[ ${#dirs[@]} == 0 ]]; then
```

Comment:
Rather than rewriting this whole script, I would expect to just add these lines later.